### ISSUE TITLE
How to encode line numbers added by the scribe to mark the place of an insertion?

### ISSUE THREAD

**Issue by ppasedach** [chchch/upama#12]
In the following examples the scribe has very carefully noted the places at which a marginal insertion is supposed to fit in:

![Selection_165](https://user-images.githubusercontent.com/11517856/128952794-26d072a9-eed6-4174-ab93-4f9d5c38ab8d.png)
![Selection_164](https://user-images.githubusercontent.com/11517856/128952796-d904f139-d6ff-4171-81bb-9b9971c3c984.png)
Not only did he use insertion marks at the places in which he forgot the akṣara which is to be inserted, also he added the line numbers behind the marginal insertions, and Xes before them. It would be nice to render these extra characters in some way, but have them ignored in the collation.

**Comment by chchch** [comment]
Oh, I've been calling that a `<note>`. See https://saktumiva.org/wiki/dravyasamuddesa/05-delhi

Although numerals are ignored automatically anyway (Punctuation -> ignore numbers).

For the Devanāgarī insertion mark there's U+A8FA, but for that we have a similar consideration as in issue #10, where we might consider using some kind of transliteration which could be rendered properly in any script, not just in Devanāgarī.

**Comment by ppasedach** [comment]
You mean something like this?

`anena ca dravyeṇa vyāḍidarśane sarve śabdā dravyābhidhāyi<add place="right margin">no<note place="inline">1</note></add><space/> bhavaṃti |`

Yes, it would be nice to also encode the insertion mark in the line. U+A8FA covers the upper half of the thing only, there's a corresponding hook below the line as well. And the X in front of the string to be inserted would be nice to have, too. I guess eventually one would need a rather long and continuously growing list of such characters, many of which might be specific to one scribe only.

**Comment by chchch** [comment]
Yeah, although the `place="inline"` part isn't really necessary; I guess I just got used to including it.

With respect to insertion marks, etc., I think that most of these characters are common to an entire tradition... I haven't seen many scribe-specific marks. So it should be fairly doable to classify and transcribe them; this would also be advantageous for studying the use of these marks across traditions — e.g., comparing how marginal additions are notated in different scripts.

I've seen some reader-specific marks though, which are used to mark-up a manuscript.

**Comment by wujastyk** [comment]
Isn't the scribal indication of the line number (7) a metamark?

**Comment by chchch** [comment]
Hmm yes perhaps it's closer to `<metamark>`, I guess I started using `<note>` because I think of numerals as "legible" characters.

**Comment by ppasedach** [comment]
`<metamark place="above and below">^</metamark><add place="bottom-margin"><metamark>✗</metamark>pratha<metamark>7</metamark></add>`
